File: data/StarCoder2/readable_format_HPEC/fortran/file130.f90
program DRB126_firstprivatesections_orig_no
    use omp_lib
    implicit none

    integer :: section_count

    section_count = 0

    call omp_set_dynamic(.FALSE.)
    call omp_set_num_threads(1)

    !$omp parallel
    !$omp sections firstprivate( section_count )
        !$omp section
        section_count = section_count+1
        print 100, section_count
        100 format ('section_count =',3i8)

        !$omp section
        section_count = section_count+1
        print 101, section_count
        101 format ('section_count =',3i8)
    !$omp end sections
    !$omp end parallel
end program

File: data/llama3_8B_ins_results/readable_format_HPEC/cpp/file130.cpp
#include <iostream>
#include <omp.h>

int main() {
    int section_count = 0;

    omp_set_dynamic(false);
    omp_set_num_threads(1);

    #pragma omp parallel
    {
        #pragma omp sections firstprivate(section_count)
        {
            #pragma omp section
            {
                section_count++;
                std::cout << "section_count = " << section_count << std::endl;
            }

            #pragma omp section
            {
                section_count++;
                std::cout << "section_count = " << section_count << std::endl;
            }
        }
    }

    return 0;
}
